webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -468,6 +485,7 @@ object DisplayNode extends Loggable {
 
         <div class="status-info col-lg-6 col-sm-5 col-xs-12">
           <h3>Status information</h3>
+          <div id={s"compliance-bar-${nodeInfo.get.id.value}"}></div>
```

Comment:
Here you should have a class "node-compliance-bar" and define all the css done below in a css file targeting that class